dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1633,3 +1633,11 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
 sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
 
 query.or.post.is.null=Request query string or post data is null.
+
+read.write.bytebuffer.null=CWWWC0009E: An attempt to read from an InputStream or write to an OutputStream failed because a variable argument ByteBuffer was null.
+read.write.bytebuffer.null.explanation=The variable argument ByteBuffer passed to the read or write API must not be null.
+read.write.bytebuffer.null.useraction=Modify the application code according to the error explanation.
+
+read.write.failed.isReady.false=CWWWC0010E: An attempt to read from an InputStream or write to an OutputStream failed because isReady API returns false.
+read.write.failed.isReady.false.explanation= If isReady returns false it is illegal to call a read or write method.
+read.write.failed.isReady.false.useraction= Modify the application code according to the error explanation.
```

Comment:
here also- can we be more specific?

Reply:
@dmuelle I don't know how to explain or be more specific about this one.  The API doc for isReady()

https://jakarta.ee/specifications/servlet/6.0/apidocs/jakarta.servlet/jakarta/servlet/servletinputstream#isReady()

this flag is dynamically changed at runtime depending on if the stream is busy writing or reading when another async resource is trying to query for isReady().

Reply:
In this case- would the operation just be retried later to see if isReady is still busy? How might the user adjust the application code to avoid the error? We are telling them to edit the app code according to the error explanation, but the explanation doesnt say anything about how to edit the code, how persistent the issue might be or what is causing it.

Reply:
Looks good, I'd suggest a slight edit to the first one

> ead.write.bytebuffer.null.useraction=Modify the application to create a non-null ByteBuffer **before using** in an InputStream or OutputStream.


What is being used in an I/O stream? the application or the ByteBuffer?

Reply:
The application needs to create a ByteBuffer which is passed into the Input/OutputStream. @dmuelle

Reply:
ok thanks- left inline suggestions for both these